core/src/main/java/org/frankframework/receivers/JavaListener.java

```diff
@@ -168,15 +167,10 @@ private Message processRequest(@Nonnull MessageWrapper<M> messageWrapper, @Nonnu
 			throw new ListenerException("JavaListener [" + getName() + "] is not opened");
 		}
 		log.debug("JavaListener [{}] processing correlationId [{}]" , getName(), messageWrapper.getCorrelationId());
-		Object object = parentSession.get(PipeLineSession.HTTP_REQUEST_KEY); //TODO dit moet weg
-		if (object != null) {
-			if (object instanceof HttpServletRequest request) {
-				ISecurityHandler securityHandler = new HttpSecurityHandler(request);
-				parentSession.setSecurityHandler(securityHandler);
-			} else {
-				log.warn("No securityHandler added for httpRequest [{}]", object::getClass);
-			}
-		}
+
+		ISecurityHandler securityHandler = new ContextSecurityHandler();
+		parentSession.setSecurityHandler(securityHandler);
```

Comment:
Ik vraag me af of dit altijd nodig is, en of het goed gaat wanneer het nodig is.

Volgens mij is dit alleen nodig wanneer een JavaListener in een apart thread wordt uitgevoerd, en niet het main-thread van het request.
En juist in dat geval weet ik niet of de Spring SecurityContextHolder de informatie heeft die nodig is.